packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -70,6 +71,16 @@ const Autocomplete = React.memo(props => {
         }
     }, [list_items, has_updating_list]);
 
+    React.useEffect(() => {
```

Comment:
We are setting the active_index and input_value in the previous useEffect, isn't it better to add this condition to that useEffect instead of writing new one in order to not set those values twice?

Reply:
I'll try on that and see if it works. Thanks @mahdiyeh-fs 🙏